src/frontend/src/components/common/Dropdown/index.tsx

```diff
@@ -8,27 +8,32 @@ const Dropdown = ({
   placeholder,
   changeHandler,
   required,
+  readOnly = false,
 }: IDropdown) => {
   return (
     <div className={`form-group ${optionalClass}`}>
       <label htmlFor={id}>{label}</label>
       <div className="dropdown">
-        <select
-          id={id}
-          className="form-control dropdown-icon-bg"
-          value={value}
-          onChange={changeHandler}
-          required={required}
-        >
-          <option key={id} value="">
-            {placeholder || 'Select one'}
-          </option>
-          {options.map((option, index) => (
-            <option key={index} value={option.value}>
-              {option.label}
+        {!readOnly ? (
+          <select
+            id={id}
+            className="form-control dropdown-icon-bg"
+            value={value}
+            onChange={changeHandler}
+            required={required}
+          >
+            <option key={id} value="">
+              {placeholder || 'Select one'}
             </option>
-          ))}
-        </select>
+            {options.map((option, index) => (
+              <option key={index} value={option.value}>
+                {option.label}
+              </option>
+            ))}
+          </select>
+        ) : (
+          value.split('_').join(' ')
```

Comment:
should we encapsulate this in a \<p\> tag? All readonly views will be a paragraph?

Reply:
Good catch! Done.